docs/pages/component/props.mdx

```diff
@@ -157,6 +157,7 @@ Adjust the control styles. This prop is need only if `controls={true}` and is an
 | hideDuration                        | boolean | The default value is `false`, allowing you to hide the duration.                            |
 | hideNavigationBarOnFullScreenMode   | boolean | The default value is `true`, allowing you to hide the navigation bar on full-screen mode.   |
 | hideNotificationBarOnFullScreenMode | boolean | The default value is `true`, allowing you to hide the notification bar on full-screen mode. |
+| hideSettingButton                   | boolean | The default value is `false`, allowing you to hide the setting button.                      |
```

Comment:
default should be true (hidden by default).
This is not a common feature and here you are bringing UI change for existing users